settings.gradle.kts

```diff
@@ -1,22 +1,22 @@
-pluginManagement {

```

Comment:
Don't touch this file

Reply:
dont believe i did, never did touch it, pretty sure github is just going mentally insane. unless im blind, the 2 things are the exact same (mightve accidently added a space somewhere somehow? i have no clue)